src/test_suite/test_runner_main.ts

```diff
@@ -63,6 +64,29 @@ export async function testRunnerMain(
     const brief = args['b'] === true;
     const markdown = args['m'] === true;
 
+    // Get the input type for the utterance we want to use for the test run
+    // from command line args. Map the value to its enum state.
+    let textType = args['t'];
+    if (textType) {
+        switch (textType.toLowerCase()) {
+            case 'scoped':
+                textType = TextType.Scoped;
```

Comment:
It is no a good idea for textType to have overloaded semantics. Initially, it is the string value of the -t command-line argument. Later, you over-write it with the string valued enumeration constant. I would keep the two concepts separate. I would probably make a Map<string,TextType> and use it as a lookup table. I would rename textType to textTypeFlag (or correctionLevelFlag) and then introduce a new variable textType: TextType (or correctionLevel: CorrectionLevel) and initialize it with the default value. Then, I'd over write that value with the value from the map, if it existed.